package.json

```diff
@@ -12,6 +12,7 @@
   "devDependencies": {
     "node-sass": "^4.14.1",
     "nuxt": "^2.14.6",
-    "sass-loader": "^10.0.3"
+    "sass-loader": "^10.0.3",
+    "webpack": "^5.1.0"
```

Comment:
With the upgrade to webpack5 it's raising `webpack/lib/RuleSet`, at least on `termux` on my smartphone.

Tomorrow I'll review it carefully, but please wait to merge it

Reply:
Did you use yarn or npm ??